### ISSUE TITLE
False positive for unchecked_use_of_nullable_value

### ISSUE THREAD

**Issue by bwilkerson** [dart-lang/sdk#40859]
If I analyze a file containing
```
void f(List<int>? list) {
  list?..length..length;
  list?..[2] = 3..[3] = 2;
}
```

I see two errors:
```
error: The expression is nullable and must be null-checked before it can be used. (unchecked_use_of_nullable_value at [nnbd_test_package] lib/test.dart:2)
error: The expression is nullable and must be null-checked before it can be used. (unchecked_use_of_nullable_value at [nnbd_test_package] lib/test.dart:3)
```
(in both cases the identifier `list` is highlighted)

I believe that the `?..` operator ought to prevent this error from being generated.

**Comment by bwilkerson** [comment]
@scheglov

**Comment by scheglov** [comment]
This was fixed in 4844fa96223e2c7af2893608328df4f604fe510a.
